samples/boards/bbc_microbit/dts/bindings/motor-controller.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Motor controller embedded in DFRobot Maqueen.
+
+compatible: "motor-controller"
+
+include: base.yaml
+
+properties:
+    reg:
+      required: true
```

Comment:
Why not just `include: i2c-device.yaml`. Is it because of the required label?

Reply:
right, forgot about it